src/adm/filter.rs

```diff
@@ -430,7 +456,12 @@ impl AdmFilter {
                     self.report(&e, tags);
                     return None;
                 }
-
+                if let Err(e) = self.check_image_hosts(img_filter, &mut tile, tags) {
+                    trace!("Rejecting tile: bad img");
+                    metrics.incr_with_tags("filter.adm.err.invalid_img_host", Some(tags));
```

Comment:
Sorry to be pedantic but one more here, especially because we already have a few metrics with the word "image" in it.
```suggestion
                    metrics.incr_with_tags("filter.adm.err.invalid_image_host", Some(tags));
```

Reply:
Nope, good catch! I had done variable refactors for the others and they missed this.